blitzstudios/sleeper-mini
# Syncronizing runtime babel packages with main sleeper app to avoid extra runtime package downloads.

### Description

In doing some profiling for minis, I noticed that loading a mini in the main Sleeper app would trigger downloads of a lot of supplemental third party packages that seemed unnecessary.

Digging into this more, it looks like some of our babel runtime packages end up being required by mini code, but was not exported to module federation in Sleeper.

This commit fixes that by synchronizing the babel runtime plugins with sleeper, ensuring that no extra downloads occur.

Now, loading either the podcast or video minis only result in about ~50kb each of data being downloaded. 

### How To Test

1. Load up the sleeper app, and load either the podcast or video minis from staging.
2. Observe log messages from `[BundleManager]`. Only two bundles should be downloaded: the main code file for the mini (`src_sleeper-podcasts_index_tsx.chunk.bundle`), and its module federation container (`sleeper-podcasts.container.bundle`);

### Screenshots/Video

Before

![Screen Shot 2023-08-07 at 2 00 00 PM](https://github.com/blitzstudios/sleeper-mini/assets/61988202/7fc5aedd-f706-4d20-b602-4c1082b72e43)

After

![Screen Shot 2023-08-07 at 1 50 41 PM](https://github.com/blitzstudios/sleeper-mini/assets/61988202/bae54150-b078-40f0-96d9-4812a966ae7e)
